src/refinement/refinement.cpp

```diff
@@ -106,12 +106,9 @@ AmrTag FirstDerivative(DevExecSpace exec_space, const ParArrayND<Real> &q,
     il = 1;
     iu = dim1 - 2;
   }
-
   Real maxd = 0.0;
-  Kokkos::parallel_reduce(
-      "refinement first derivative",
-      Kokkos::MDRangePolicy<Kokkos::Rank<3>>(
-          exec_space, {kl, jl, il}, {ku + 1, ju + 1, iu + 1}, {1, 1, iu + 1 - il}),
+  pmb->par_for(
```

Comment:
I don't love that this still compiles, but anyway, this should be `par_reduce`

```suggestion
  pmb->par_reduce(
```